src/patternfly/components/Button/examples/button-states-example.hbs

```diff
@@ -43,18 +46,32 @@
 {{#> button button--modifier="pf-m-danger pf-m-active"}}
   Danger Active
 {{/button}}
-{{#> button button--modifier="pf-m-danger pf-m-disabled"}}
+{{#> button button--modifier="pf-m-danger" button--attribute="disabled"}}
   Danger Disabled
 {{/button}}
-{{#> button button--modifier="pf-m-link" button--HasPlusCircleIcon="true"}}
+<br><br>
+{{#> button button--modifier="pf-m-link"}}
+  {{> button-icon button-icon--type="plus-circle" button-icon--modifier="pf-c-button__icon"}}
   Link button
 {{/button}}
-{{#> button button--modifier="pf-m-link pf-m-focus" button--HasPlusCircleIcon="true"}}
+{{#> button button--modifier="pf-m-link pf-m-focus"}}
+  {{> button-icon button-icon--type="plus-circle" button-icon--modifier="pf-c-button__icon"}}
   Link button
 {{/button}}
-{{#> button button--modifier="pf-m-link pf-m-active" button--HasPlusCircleIcon="true"}}
+{{#> button button--modifier="pf-m-link pf-m-active"}}
+  {{> button-icon button-icon--type="plus-circle" button-icon--modifier="pf-c-button__icon"}}
   Link button
 {{/button}}
-{{#> button button--modifier="pf-m-link pf-m-disabled" button--HasPlusCircleIcon="true"}}
+{{#> button button--modifier="pf-m-link" button--attribute="disabled"}}
+  {{> button-icon button-icon--type="plus-circle" button-icon--modifier="pf-c-button__icon"}}
   Link button
+{{/button}}
+<br><br>
+{{#> button button--modifier="pf-m-plain" button--HasTimesIcon="true" button--attribute='aria-label="Remove"'}}
```

Comment:
this could be a new issue but should we have these handlebar files for specific icons? We could just use the `button-icon.hbs` file couldn't we?

Reply:
Yep! I created an issue for that when I started this change https://github.com/patternfly/patternfly-next/issues/927

Reply:
:)